### PR TITLE
Add macro to match the adapter.

### PR DESCRIPTION
**Pull request by fokinv**

### CODE REVIEW

File: components/net/bluetooth_thread.rs [hunk 1]
```diff
@@ -351,10 +359,8 @@ impl BluetoothManager {
     fn request_device(&mut self,
                       options: RequestDeviceoptions,
                       sender: IpcSender<BluetoothResult<BluetoothDeviceMsg>>) {
-        let mut adapter = match self.get_or_create_adapter() {
-            Some(a) => a,
-            None => return drop(sender.send(Err(String::from(ADAPTER_ERROR)))),
-        };
+        let mut adapter = return_or_get_adapter!(self, sender);
+
```
Comment (dati91) [inline components/net/bluetooth_thread.rs]: remove the empty line

File: components/net/bluetooth_thread.rs [hunk 2]
```diff
@@ -48,6 +48,14 @@ macro_rules! return_if_cached(
     );
 );
 
+macro_rules! return_or_get_adapter(
+    ($bl_manager:expr, $sender:expr) => (
+        match $bl_manager.get_or_create_adapter() {
+            Some(adapter) => adapter,
+            None => return drop($sender.send(Err(String::from(ADAPTER_ERROR)))),
+        })
```
Comment (dati91) [inline components/net/bluetooth_thread.rs]: the `)` should be in the next line, and a `;` is missing

**Comment by dati91** [comment]
lgtm
@mmatyas r?

File: components/net/bluetooth_thread.rs [hunk 3]
```diff
@@ -48,6 +48,15 @@ macro_rules! return_if_cached(
     );
 );
 
+macro_rules! return_or_get_adapter(
```
Comment (mmatyas) [inline components/net/bluetooth_thread.rs]: I'd prefer `get_adapter_or_return`, or `..._return_error`.

**Comment by mmatyas** [comment]
Other than that, LGTM.

**Pull request merged**